src/components/TasksListItem/TasksListItem.tsx

```diff
@@ -66,32 +66,36 @@ export const TasksListItem = ({
           </div>
           <ChevronDownFill
             size={16}
-            className='Layer__tasks__expand-icon'
+            className="Layer__tasks__expand-icon"
             style={{
-              transform: isOpen ? 'rotate(0deg)' : 'rotate(-180deg)',
+              transform: isOpen ? "rotate(0deg)" : "rotate(-180deg)",
             }}
           />
         </div>
         <div className={taskBodyClassName}>
-          <div className='Layer__tasks-list-item__body-info'>
+          <div className="Layer__tasks-list-item__body-info">
             <Text size={TextSize.sm}>{task.question}</Text>
-            {task.user_response_type === 'FREE_RESPONSE' ? (
+            {task.user_response_type === "FREE_RESPONSE" ? (
               <Textarea
                 value={userResponse}
                 onChange={(e: React.ChangeEvent<HTMLTextAreaElement>) =>
                   setUserResponse(e.target.value)
                 }
               />
             ) : null}
-            <div className='Layer__tasks-list-item__actions'>
-              {task.user_response_type === 'UPLOAD_DOCUMENT' ? (
+            <div className="Layer__tasks-list-item__actions">
+              {task.user_response_type === "UPLOAD_DOCUMENT" ? (
                 <FileInput
-                  onUpload={(file: File) => {
-                    uploadDocumentForTask(task.id, file)
-                    setIsOpen(false)
-                    goToNextPageIfAllComplete(task)
+                  onUpload={async (file: File) => {
+                    try {
+                      await uploadDocumentForTask(task.id, file);
+                      setIsOpen(false);
+                      goToNextPageIfAllComplete(task);
+                    } catch (e) {
+                      // If upload fails... no-op
+                    }
```

Comment:
Only substantive change in this file

Reply:
(Nit - no need to change)

I'm mildly in favor of calling `console.error("<Descriptive message>", e)` when these things happens.